### PR TITLE
Route to chatroom dynamically from the repo name clicked on Dashboard

### PR DESCRIPTION
**Pull request by anicknam**
Outstanding issues:
1) Logout
2) Link to Dashboard is not showing on the Chatroom

### CODE REVIEW

File: client/api/chatroom/messageRequest.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ function _get(url) {
 }
 
 function getMessages(chatroomid) {
+  console.log(chatroomid)
```
Comment (chasestarr) [inline client/api/chatroom/messageRequest.js]: Could you please remove this console log?

File: client/components/chatroom.js [hunk 2]
```diff
@@ -33,10 +37,12 @@ class Chatroom extends React.Component {
     .then((data) => {
       const username = JSON.parse(data).username;
       const userAvatarUrl = JSON.parse(data)._json.avatar_url;
+      console.log(username, userAvatarUrl);
       this.setState({ 
         username: username,
         userAvatarUrl: userAvatarUrl
       });
+      console.log(this.state);
```
Comment (chasestarr) [inline client/components/chatroom.js]: Again, please remove console logs

File: client/components/entermessage.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ class EnterMessage extends React.Component {
     super(props);
     this.state = {
       value: '',
-      chatroom: 'anicknam/gittalk' // FIXME make me dynamic
+      chatroom: this.props.username + '/' + this.props.reponame // FIXME make me dynamic
```
Comment (chasestarr) [inline client/components/entermessage.js]: Is the comment at the end of this line still necessary? Please remove if not

File: client/components/messages.js [hunk 4]
```diff
@@ -35,11 +35,14 @@ class Messages extends React.Component {
 
   componentDidMount() {
   // fetch all messages from DB
-    // getMessages('anicknam/gittalk')
-    // .then(messages => {
-    //   this.setState({ messages: JSON.parse(messages) });
-    // })
-    // .catch(err => console.log(err));
+    console.log('messages mount', this.props);
```
Comment (chasestarr) [inline client/components/messages.js]: :dog:

**Comment by tankwan** [comment]
🔥

**Pull request merged**